addons/website_slides/controllers/main.py

```diff
@@ -754,6 +770,18 @@ def slide_quiz_submit(self, slide_id, answer_ids):
             'quizAttemptsCount': quiz_info['quiz_attempts_count'],
             'rankProgress': rank_progress,
         }
+    
+    @http.route(['/slides/slide/quiz/save_slide_answsers'], type='json', auth='public', website=True)
+    def slide_save_answsers(self, quizAnswers):
+        if 'slide_answer_quiz' not in request.session:
+            request.session['slide_answer_quiz'] = json.dumps({})
+        session_slide_answer_quiz = json.loads(request.session['slide_answer_quiz'])
+        values={}
```

Comment:
You need to have spaces between an operator
So `values = {}`

Your flake8 linter should warn you if correctly configured (https://github.com/odoo/enterprise/wiki/Welcome)

Please check the whole commit for those.